20210325/AGENDA.md

```diff
@@ -0,0 +1,59 @@
+# Sardana Follow-up Meeting - 2021/03/25
+
+To be held on 2021/03/25 at 14:00
+
+## Agenda
+
+1. Discussion about Questionnaire ([#24](https://github.com/sardana-org/sardana-followup/issues/24))
```

Comment:
To start with, I can make a short demo on the fresh feature (merged last night:)) of the [H5 session](https://sardana-controls.org/users/scan.html#nxscanh5-filerecorder) which also adds support to the SWMR mode. I will add it to the agenda, but if there is no interest we could simply skip it.